### ISSUE TITLE
2.2 release

### ISSUE THREAD

**Issue by samrocketman** [jenkinsci/slack-plugin#296]
This issue is to track Slack plugin 2.2 release.

**Comment by samrocketman** [comment]
https://github.com/jenkinsci/slack-plugin/pull/295 - update CHANGELOG

**Comment by samrocketman** [comment]
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-release-plugin:2.5:prepare (default-cli) on project slack: Unable to commit files
[ERROR] Provider message:
[ERROR] The git-push command failed.
[ERROR] Command output:
[ERROR] remote: error: GH006: Protected branch update failed for refs/heads/master.
[ERROR] remote: error: Required status check "Jenkins" is expected.
[ERROR] To ssh://git@github.com/jenkinsci/slack-plugin.git
[ERROR] ! [remote rejected] master -> master (protected branch hook declined)
[ERROR] error: failed to push some refs to 'ssh://git@github.com/jenkinsci/slack-plugin.git'
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

@kmadel the settings you enabled on this repository requiring PRs breaks the ability to release.

**Comment by samrocketman** [comment]
Release is blocked by repository settings.

**Comment by kmadel** [comment]
Repository settings have been updated. Although this never blocked me when doing a release following your list of steps.

**Comment by jiamengyu** [comment]
Can you please include PR #299 into release 2.2?  I just complete the test locally. I really want proxy issue resolved so we can use Slack notification in enterprise environment.

**Comment by samrocketman** [comment]
@jiamengyu sure, I haven't released yet.  Can you squash your commits into a single commit?

**Comment by jiamengyu** [comment]
@samrocketman  I have squashed all comments into one as Proxy fix #102.  Please let me know if anything else I need to do.

**Comment by samrocketman** [comment]
2.2 released.  It can be downloaded from https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/slack/2.2/.  It typically takes roughly 4-8 hours to show up in the update center.

Changes in this released can be viewed in the [CHANGELOG](https://github.com/jenkinsci/slack-plugin/blob/master/CHANGELOG.md).
